execution/gethexec/sequencer.go

```diff
@@ -490,7 +490,7 @@ func (s *Sequencer) preTxFilter(_ *params.ChainConfig, header *types.Header, sta
 	return nil
 }
 
-func (s *Sequencer) postTxFilter(header *types.Header, _ *arbosState.ArbosState, tx *types.Transaction, sender common.Address, dataGas uint64, result *core.ExecutionResult) error {
+func (s *Sequencer) postTxFilter(header *types.Header, _ *state.StateDB, _ *arbosState.ArbosState, tx *types.Transaction, sender common.Address, dataGas uint64, result *core.ExecutionResult) error {
```

Comment:
Shouldn't this reject the tx if IsTxFiltered?

Reply:
I didn't know we wanted to currently enforce this on the sequencer side, if yes, then I'll change it add a check

Reply:
cool